Header header

uint32  distance
uint16  confidence
uint16  transmit_duration
uint32  ping_number
uint32  min_range
uint32  range_length
uint32  gain_setting

uint8[] ping_data
